src/models/index.d.ts

```diff
@@ -0,0 +1,160 @@
+interface IInvokeConfig {
+    params: { apiVersion: string; endpoint?: string; };
+    functions: { plateGen: { name: string } };
+}
+
+interface IMOTConfig {
+    endpoint: string;
+    documentDir: "CVS";
+    documentNames: {
+        vtg6_vtg7: "VTG6_VTG7.pdf";
+    };
+    api_key: string;
+}
+
+interface IS3Config {
+    endpoint: string;
+}
+
+interface ICertificatePayload {
+    Watermark: string;
+    PLATES_DATA?: any;
+}
+
+interface IGeneratedCertificateResponse {
+    fileName: string;
+    vin: string;
+    vrm: string;
+    dateOfIssue: string;
+    certificateType: string;
+    fileFormat: string;
+    fileSize: string;
+    certificate: Buffer;
+    email: string;
+}
+
+interface ITechRecordWrapper {
+    primaryVrm: string;
+    secondaryVrms?: string[];
+    vin: string;
+    systemNumber: string;
+    partialVin?: string;
+    trailerId: string;
+    techRecord: ITechRecord[];
+}
+
+interface ITechRecord {
+    createdAt: string;
+    createdByName: string;
+    createdById: string;
+    lastUpdatedAt: string;
+    lastUpdatedByName: string;
+    lastUpdatedById: string;
+    updateType: string;
+    reasonForCreation: string;
+    statusCode: string;
+    vehicleType: string;
+    plates: IPlates;
+    brakes: {
```

Comment:
Is it worth trying to split them out?
We know the payload might change depending on the type of vehicle we have.
We could have a common interface we extend.